IdentityCore/src/MSIDJsonSerializableFactory.h

```diff
@@ -23,20 +23,31 @@
 
 #import <Foundation/Foundation.h>
 
-@class MSIDBrokerOperationRequest;
 @protocol MSIDJsonSerializable;
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface MSIDBrokerOperationRequestFactory : NSObject
+@interface MSIDJsonSerializableFactory : NSObject
 
-+ (void)registerOperationRequestClass:(Class<MSIDJsonSerializable>)operationRequestClass
-                            operation:(NSString *)operation;
+/*!
+ Bind class type with sepcifc class type key in json paylod.
+ This method is tread safe.
+ */
++ (void)registerClass:(Class<MSIDJsonSerializable>)value forKey:(NSString *)key;
```

Comment:
Since this can be used in all over the project, will it be better to create a dedicated constant file for possible keys used here, as an effort to better manage the keys and avoid key conflicts. What do you think? :)

Reply:
Good point, will do.

Reply:
done